scripts/tests/provisioning/migrate_vm_to_safemode.expect

```diff
@@ -83,7 +83,7 @@ expect {
 
 set timeout 500
 expect {
-	"$USER@recovery:/#" {}
+	"root@recovery:/#" {}
```

Comment:
Can we just re-set USER to root and use it?

Reply:
Well, I don't think there's a reason to variablize it, since the recovery USB will *never* use `admin` as its login (because it necessarily doesn't include NIAuth).